### DISCUSSION TITLE
Texture Shortcut Folders

### DISCUSSION THREAD

**Discussion by smorgan134** [kubilus1/autoortho#189] [General]
I've noticed that after the .zip files are extracted, the textures in each z_xxx_## folder get copied to \z_autoortho\_textures folder, and it appears a link or shortcut gets created.  However, the link/shortcut doesn't actually point to anything.  In my mind the texture link/shortcut in each z_xxx_## folder should point to \z_autoortho\_textures, but when the link/shortcut is created, it points to \z_autoortho\textures, which technically doesn't exist.  When I click on the texture link/shortcut in the z_xxx_## folders, I get an error stating the folder can't be found (which is expected behavior considering it doesn't point to anything).  Maybe this is intentional.  It doesn't appear to be causing an issue with the tiles loading, but I thought I would bring it up anyway.

**Comment by kubilus1** [comment]
The `z_autoortho\textures` folder is a mount point and this tool mounts to that location in order to detect DDS requests.  The _textures folder exist and contains stuff since the little water transparency files from ortho4xp need to be preserved somewhere.  

https://kubilus1.github.io/autoortho/details/

**Reply by smorgan134** [reply]
Thanks for the clarification.  The tool is working really well.  I did a flight with it this morning with no issues.  Thanks for taking your time to develop this.  It is a huge time and space saver to not have to download Ortho all the time.

**Comment by kubilus1** [comment]
Great!  Glad it's working for you.
